e2e/app/page/workspace-edit-page.ts

```diff
@@ -279,12 +281,16 @@ export default class WorkspaceEditPage extends AuthenticatedPage {
 
   async isLoaded(): Promise<boolean> {
     try {
-      await this.waitUntilTitleMatch(PAGE.TITLE);
-      await this.getWorkspaceNameTextbox();
-      await new SelectMenu(this.page, LABEL_ALIAS.SELECT_BILLING).getSelectedValue();
-      await this.getCreateWorkspaceButton();
+      await Promise.all([
+        waitForDocumentTitle(this.page, PAGE.TITLE),
```

Comment:
will it be ok if the request will happen in any order?

Reply:
yes. what matters is that they're all present.